internal/api/grpc/user/query.go

```diff
@@ -7,18 +7,22 @@ import (
 	user_pb "github.com/zitadel/zitadel/pkg/grpc/user"
 )
 
-func UserQueriesToQuery(queries []*user_pb.SearchQuery) (_ []query.SearchQuery, err error) {
+func UserQueriesToQuery(queries []*user_pb.SearchQuery, level uint8) (_ []query.SearchQuery, err error) {
```

Comment:
would love to test the new level argument, but I can't find any existing unit tests for `UserQueriesToQuery`.

Reply:
As we are in the process of moving the APIs anyway, we will cover the functionality with unit tests and integrations tests as soon as possible. For an example you can have a look at https://github.com/zitadel/zitadel/tree/main/internal/api/grpc/user/v2 , if you want to still cover it with unit tests, be my guest.